<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NMS Crafting</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="lQuantite">
          <property name="font">
           <font>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Quantité</string>
          </property>
          <property name="buddy">
           <cstring>sbQuantite</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lX">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>10</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>x</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="pbRecherche">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Lister Ingrédients</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="lRecette">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>Recette</string>
          </property>
          <property name="buddy">
           <cstring>qcbListeRecettes</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="sbQuantite">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="toolTip">
           <string>Quantité à produire</string>
          </property>
          <property name="statusTip">
           <string>Quantité à produire</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="qcbListeRecettes">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="maxVisibleItems">
           <number>20</number>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::NoInsert</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="cbFarming">
        <property name="toolTip">
         <string>Pour afficher le nombre de plant(s) et dome(s) nécessaire</string>
        </property>
        <property name="statusTip">
         <string>Active les données de farming</string>
        </property>
        <property name="text">
         <string>Farming ?</string>
        </property>
        <property name="icon">
         <iconset resource="icones.qrc">
          <normaloff>:/farming_24.png</normaloff>:/farming_24.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="imageRecette">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>15</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Image</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lDescription">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>15</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTextEdit" name="teRecetteDescription">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeView" name="vue">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="animated">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <attribute name="headerMinimumSectionSize">
         <number>50</number>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>180</number>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pbExpand">
          <property name="text">
           <string>Déployer Tout</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbCollapse">
          <property name="text">
           <string>Réduire Tout</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbAutoExpand">
          <property name="toolTip">
           <string>Etend automatiquement la liste des ingrédients</string>
          </property>
          <property name="statusTip">
           <string>Activer l'étendu automatique de la liste</string>
          </property>
          <property name="text">
           <string>Etendre tout automatiquement ?</string>
          </property>
          <property name="icon">
           <iconset resource="icones.qrc">
            <normaloff>:/expand_24.png</normaloff>:/expand_24.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbTest">
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>T</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="mbMenuPrincipal">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="mFichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="aQuitter"/>
   </widget>
   <widget class="QMenu" name="mEditer">
    <property name="title">
     <string>&amp;Edition</string>
    </property>
    <addaction name="aAjouterRecette"/>
   </widget>
   <widget class="QMenu" name="mOutil">
    <property name="title">
     <string>&amp;Outil</string>
    </property>
    <addaction name="aPreferences"/>
    <addaction name="separator"/>
    <addaction name="aRestoreRecipe"/>
    <addaction name="aRestaurerTaillePosition"/>
    <addaction name="separator"/>
    <addaction name="aAutoExpand"/>
    <addaction name="aFarming"/>
   </widget>
   <widget class="QMenu" name="mAide">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="aVerifMiseAJour"/>
    <addaction name="aAPropos"/>
   </widget>
   <addaction name="mFichier"/>
   <addaction name="mEditer"/>
   <addaction name="mOutil"/>
   <addaction name="mAide"/>
  </widget>
  <action name="aQuitter">
   <property name="icon">
    <iconset resource="icones.qrc">
     <normaloff>:/close_24.png</normaloff>:/close_24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="toolTip">
    <string>Fermer l'application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="aAjouterRecette">
   <property name="icon">
    <iconset resource="icones.qrc">
     <normaloff>:/add_24.png</normaloff>:/add_24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Ajouter recette...</string>
   </property>
  </action>
  <action name="aPreferences">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icones.qrc">
     <normaloff>:/settings_24.png</normaloff>:/settings_24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Préférences...</string>
   </property>
  </action>
  <action name="aFarming">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icones.qrc">
     <normaloff>:/farming_24.png</normaloff>:/farming_24.png</iconset>
   </property>
   <property name="text">
    <string>Afficher données de farm</string>
   </property>
  </action>
  <action name="aAutoExpand">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icones.qrc">
     <normaloff>:/expand_24.png</normaloff>:/expand_24.png</iconset>
   </property>
   <property name="text">
    <string>Etendre automatiquement</string>
   </property>
  </action>
  <action name="aRestoreRecipe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icones.qrc">
     <normaloff>:/memorize_24.png</normaloff>:/memorize_24.png</iconset>
   </property>
   <property name="text">
    <string>Restaurer dernière recette</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="aRestaurerTaillePosition">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icones.qrc">
     <normaloff>:/size_24.png</normaloff>:/size_24.png</iconset>
   </property>
   <property name="text">
    <string>Restaurer taille &amp;&amp; position</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="aAPropos">
   <property name="icon">
    <iconset resource="icones.qrc">
     <normaloff>:/about_24.png</normaloff>:/about_24.png</iconset>
   </property>
   <property name="text">
    <string>A &amp;Propos...</string>
   </property>
  </action>
  <action name="aVerifMiseAJour">
   <property name="icon">
    <iconset resource="icones.qrc">
     <normaloff>:/update_24.png</normaloff>:/update_24.png</iconset>
   </property>
   <property name="text">
    <string>Vérifier les mises à jour</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>sbQuantite</tabstop>
  <tabstop>qcbListeRecettes</tabstop>
  <tabstop>pbRecherche</tabstop>
  <tabstop>pbExpand</tabstop>
  <tabstop>pbCollapse</tabstop>
  <tabstop>vue</tabstop>
 </tabstops>
 <resources>
  <include location="icones.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbExpand</sender>
   <signal>clicked()</signal>
   <receiver>vue</receiver>
   <slot>expandAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCollapse</sender>
   <signal>clicked()</signal>
   <receiver>vue</receiver>
   <slot>collapseAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
